components/feature/sitepermissions/src/main/java/mozilla/components/feature/sitepermissions/SitePermissionsFeature.kt

```diff
@@ -596,6 +604,17 @@ class SitePermissionsFeature(
                     shouldSelectRememberChoice = true
                 )
             }
+            is ContentMediaKeySystemAccess -> {
+                createSinglePermissionPrompt(
+                    context,
+                    host,
+                    permissionRequest,
+                    R.string.mozac_feature_sitepermissions_media_key_system_access_title,
+                    R.drawable.mozac_ic_video,
```

Comment:
I think we need to change this icon as this is used by the camera permission, could cause some confusing, maybe we could use audio or link(in the Android Section)  from https://design.firefox.com/icons/viewer/ as they look similar to the icon in desktop

What desktop is using 
<img width="360" alt="image" src="https://user-images.githubusercontent.com/773158/101081067-9315ec80-3577-11eb-9a05-de741cdf6178.png">


Link
<img width="134" alt="image" src="https://user-images.githubusercontent.com/773158/101080931-6b268900-3577-11eb-9b1a-838b9c4126ec.png">

Audio
<img width="133" alt="image" src="https://user-images.githubusercontent.com/773158/101080996-7ed1ef80-3577-11eb-88d6-47de71876e11.png">